gym_xiangqi/xiangqi_game.py

```diff
@@ -55,10 +62,13 @@ def on_init(self, agent_piece, enemy_piece):
         return True
 
     def init_board(self):
+        '''
+        Initializes Board() and load the board image
+        '''
         # set board_background
         board = Board().board_background
         board = pygame.transform.scale(
-            board, (self.boardWidth, self.boardHeight)
+            board, (Board().boardWidth, Board().boardHeight)
```

Comment:
I think we can call `Board()` once and save that instead of calling `Board()` constructor multiple times.

Reply:
fixed!